packages/rx-utils/src/live-data.utils.ts

```diff
@@ -26,13 +30,50 @@ const tupleC = <A>(a: A) => <B>(b: B): [A, B] => [a, b];
 const combineRD = <EA, A, EB, B>(fa: RemoteData<EA, A>, fb: RemoteData<EB, B>): RemoteData<EA | EB, [A, B]> =>
 	remoteData.ap<EA | EB, B, [A, B]>(remoteData.map(fa, tupleC), fb);
 
-export const instanceLiveData: MonadThrow2<URI> & CoproductLeft<URI> = {
+export const instanceLiveData: MonadThrow2<URI> & CoproductLeft<URI> & Filterable2<URI> = {
 	URI,
 	...getApplicativeComposition(instanceObservable, remoteData),
 	chain: (fa, f) => instanceObservable.chain(fa, a => (isSuccess(a) ? f(a.value) : of(a))),
 	throwError: e => of(failure(e)),
 	coproductLeft: <EA, A, EB, B>(fa: LiveData<EA, A>, fb: LiveData<EB, B>) =>
 		instanceObservable.map(combineLatest(fa, fb), ([a, b]) => combineRD(a, b)),
+	filter: <E, A>(fa: LiveData<E, A>, p: Predicate<A>): LiveData<E, A> =>
```

Comment:
👍
was missing it so much

Reply:
> 👍
> was missing it so much

note that it's different from usual `filterRD` - it passes inner RemoteData as is if not RemoteSuccess

Reply:
that is just what i need. i wrote a helpers yesterday with exact behaviour :)